Tests/Docker/StorageApiWriterTest.php

```diff
@@ -940,9 +990,10 @@ public function testWriteTableOutputMappingWithEmptyStringPk()
             ]
         );
         $this->assertFalse($handler->hasWarningThatContains("Output mapping does not match destination table: primary key '' does not match '' in 'out.c-docker-test.table9'."));
```

Comment:
dal bych tam teda enem $this->assertFalse($handler->hasWarningThatContains("Output mapping does not match destination table"));